src/Swal.php

```diff
@@ -15,6 +17,7 @@
  */
 class Swal
 {
+    public const SESSION_KEY = 'sweetalert2';
```

Comment:
After refactoring the session key to be a const, I don't like the string value anymore 😄 

Let's make it `'sweetalert2-message'` or something similar?

Reply:
it would look better here:

```js
window.addEventListener('sweetalert2-message', async (event) => {
```

Reply:
```suggestion
    public const SESSION_KEY = 'sweetalert2-message';
```